### PR TITLE
Server testing setup and test example

### PR DESCRIPTION
**Pull request by PaulKovalov**
Here I create a setup for testing - I will add better tests later, for now just setting up. Setting the base branch to `server-setup` for cleaner diff. When merging, will change to master.

### CODE REVIEW

File: portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  public void setUp() {
+    servlet = new CommentsServlet();
+    request = new MockHttpServletRequest();
+    response = new MockHttpServletResponse();
+    helper.setUp();
+  }
+
+  @Test
+  void testPost() {
+    Gson gson = new Gson();
```
Comment (PaulKovalov) [inline portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:30]: This is not used here, forgot to remove it. It is removed in the further PRs though, so I hope it is okay

File: portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Comment comment = new Comment();
```
Comment (PaulKovalov) [inline portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:31]: This is not used here, forgot to remove it. It is removed in the further PRs though, so I hope it is okay

File: portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    request.setMethod("POST");
+    request.setContentType("text/html");
+    request.addParameter("username", "paul");
+    request.addParameter("text", "A nice comment");
+    try {
+      servlet.doPost(request, response);
+      assertEquals(response.getStatus(), 200);
```
Comment (harry-7) [inline portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:38]: Are there cases where the server throws an error? If yes, then we should be testing them too.

Comment (PaulKovalov) [reply portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:38]: There are cases when server throws exception, yes. Do you mean I have to check that the correct exception was thrown?

Comment (harry-7) [reply portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:38]: Nah, I was talking about the response status returned. You also have to check for failure cases in a test.

Comment (PaulKovalov) [reply portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:38]: Sure, this PR is some basic test setup, in branches `writing-server-tests` and `more-server-tests` I do more thorough testing. Here is setup only and an example of how Im going to write tests.

Comment (harry-7) [reply portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java:38]: Ah, I see. okay.

**Pull request merged**